src/components/Event/Event.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { Text } from '../Text';
+import { EVENT_DATA_HOOKS, EVENT_DATA_KEYS } from './dataHooks';
+import styles from './Event.st.css';
+
+export interface EventProps {
+  time?: string;
+  title: string;
+  showTime?: boolean;
+  multiday?: boolean;
+  selected?: boolean;
+}
+
+interface DefaultProps {
+  showTime: boolean;
+  multiday: boolean;
+  selected: boolean;
+  'data-hook': string;
+}
+
+/** Event */
+export class Event extends React.Component<EventProps> {
+  static displayName = 'Event';
+  static defaultProps: DefaultProps = {
+    showTime: true,
+    multiday: false,
+    selected: false,
+    'data-hook': EVENT_DATA_HOOKS.Event,
```

Comment:
We actually don't want a default value here, because then consumers will start using it, and we won't be able to change `EVENT_DATA_HOOKS.Event`, or else we'll break their app